### PR TITLE
adding missing ImagePullSecrets section to web deployment

### PR DESCRIPTION
**Pull request by GerardoMoraS**
## What was changed

There was missing ImagePullSecrets section in Web Deployment please check following [ticket](https://github.com/temporalio/helm-charts/issues/419).
## Checklist

1. Closes 
[[Feature Request] Support passing ImagePullSecrets to Temporal Web Deployment Helm Template](https://github.com/temporalio/helm-charts/issues/419)
2. How was this tested:

By adding image pull secrets in development environment and verifying that cluster now pulls images from our private registry.
3. Any docs updates needed?

### CODE REVIEW

File: charts/temporal/templates/server-deployment.yaml [hunk 1]
```diff
@@ -55,7 +55,7 @@ spec:
       initContainers:
         {{- if $.Values.cassandra.enabled }}
         - name: check-cassandra-service
-          image: busybox
+          image:   "{{ $.Values.busyBox.image.repository }}:{{ $.Values.busyBox.image.tag }}"
```
Comment (robholland) [inline charts/temporal/templates/server-deployment.yaml:58]: Please remove this change, it's unrelated to the image pull secrets.

File: charts/temporal/values.yaml [hunk 2]
```diff
@@ -5,6 +5,13 @@ fullnameOverride: ""
 # (eg. disable helm hook delete policy)
 debug: false
 
+imagePullSecrets: {}
+
+busyBox:
```
Comment (robholland) [inline charts/temporal/values.yaml:10]: Please remove this change.

**Review by robholland — CHANGES_REQUESTED** [review]

**Comment by robholland** [comment]
Please sign the CLA.

File: charts/temporal/templates/server-deployment.yaml (hunk 1 above)
Comment (GerardoMoraS) [reply charts/temporal/templates/server-deployment.yaml]: done

File: charts/temporal/values.yaml (hunk 2 above)
Comment (GerardoMoraS) [reply charts/temporal/values.yaml]: done

File: charts/temporal/templates/server-deployment.yaml [hunk 3]
```diff
@@ -55,7 +55,7 @@ spec:
       initContainers:
         {{- if $.Values.cassandra.enabled }}
         - name: check-cassandra-service
-          image: busybox
+          image: busy
```
Comment (robholland) [inline charts/temporal/templates/server-deployment.yaml:58]:
```suggestion
          image: busybox
```

**Review by robholland — CHANGES_REQUESTED** [review]

File: charts/temporal/templates/server-deployment.yaml (hunk 3 above)
Comment (GerardoMoraS) [reply charts/temporal/templates/server-deployment.yaml]: yeah I just fixed the typo

**Review by robholland — APPROVED** [review]
Thanks for your contribution :)

**Pull request merged**